test/test_turbomole.py

```diff
@@ -0,0 +1,231 @@
+                                     -1.656479e-14, 0.009036918])
+
+
+
+
+
+
+
+        derivative_coupling01 = np.array([0.0, 6.959001e-11, 0.0, 0.0, 8.935593e-11, 0.0, 
+                                                0.0, -0.1983295, 0.0, 0.0, 0.1983295, 0.0, 0.0, 
+                                               -0.1983295, 0.0, 0.0, 0.1983295, 0.0])
+
+
+        derivative_coupling02 = np.array([0.0, 0.008271754, 0.0, 0.0, 0.008271754, 0.0, 0.0, 
+                                               -0.02639676, 0.0, 0.0, -0.02639676, 0.0, 0.0, -0.02639676, 
+                                                0.0, 0.0, -0.02639676, 0.0])
+
+
+
+
+
+        derivative_coupling03 = np.array([0.0, -2.847169e-12, 0.0, 0.0, 2.998042e-12, 0.0, 0.0, 0.008101865, 
+                                         0.0, 0.0, -0.008101865, 0.0, 0.0, 0.008101865, 0.0, 0.0, -0.008101865, 0.0])
+
+
+
+
+
+        derivative_coupling12 = np.array([0.6964976, 0.0, -2.620811e-10, 0.6964976, 0.0, 2.889253e-10, 
+                                             -0.3479412, 0.0, -0.1161213, -0.3479412, 0.0, 0.1161213, -0.3479412, 0.0, 0.1161213, -0.3479412, 0.0, -0.1161213])
+
+
+        derivative_coupling13 = np.array([-1.939935e-10, 0.0, -0.008676887, -1.562714e-10, 0.0, 0.008676887, 
+                                              0.08383228, 4.645253e-13, -0.008070732, -0.08383228, -4.639483e-13, 
+                                             -0.008070731, 0.08383228, 4.341767e-13, 0.008070732, -0.08383228, -4.308535e-13, 0.008070731]) 
+
+
+        derivative_coupling23 = np.array([-3.254183, 0.0, -4.535725e-10, -3.254183, 0.0, 9.549593e-11, 1.640597, 
+                                               0.0, 0.5104488, 1.640597, 0.0, -0.5104488, 1.640597, 0.0, -0.5104488, 1.640597, 0.0, 0.5104488])
+
+
+
+
+        gs_e_from_ridft = model.energies[0] 
+
+        ex_e_1_from_egrad = model.energies[1]
+        ex_e_2_from_egrad = model.energies[2]
+
+
+        gs_grad_from_rdgrad = model.gradients[0] 
+
+
+        ex_st_1_gradients_from_egrad = model.gradients[1]
+
+
+        ex_st_2_gradients_from_egrad = model.gradients[2]
+
+        
+
+        derivative_coupling01_from_egrad = model.derivative_coupling[0][1]
+        derivative_coupling02_from_egrad = model.derivative_coupling[0][2] 
+        derivative_coupling03_from_egrad = model.derivative_coupling[0][3] 
+        derivative_coupling12_from_egrad = model.derivative_coupling[1][2]
+        derivative_coupling13_from_egrad = model.derivative_coupling[1][3]
+        derivative_coupling23_from_egrad = model.derivative_coupling[2][3]
+
+        self.assertAlmostEqual(gs_energy, gs_e_from_ridft, places=10)
+
+        self.assertAlmostEqual(excited_1_energy, ex_e_1_from_egrad, places=10)
+        self.assertAlmostEqual(excited_2_energy, ex_e_2_from_egrad, places=10)
+
+        np.testing.assert_almost_equal(gs_gradients, gs_grad_from_rdgrad,decimal = 10)
```

Comment:
Do these `np.testing` functions work with `pytest` the way you're expecting? I have no idea so I'm genuinely asking.